### ISSUE TITLE
getLatestReferringParams returns the previous link after app close/app

### ISSUE THREAD

**Issue by willlaserlike** [BranchMetrics/react-native-branch-deep-linking#282]
I'm having this issue with my react native app on iOS . The same code works perfectly on Android.  Here are the steps to reproduce it on iOS.
1. Click on a branch link. The app opens and getLatestReferringParams() returns the correct link data
2. Press home button to let app go to background. (I tried wait here from 10s to 1 min. It doesn't matter)
3. Click on app icon to bring it back to foreground. getLatestReferringParams() returns the same link data as in step 1. It should return false for clicked_branch_link.
4. Repeat 2 and 3. getLatestReferringParams() always returns false as it should.

In summary, after opening the app through a branch link, the branch link data is kept for the next session, which it should clean upon each new session. 

Please advice is it a known issue and what's a workaround for this?

Thanks!

**Comment by sequoiaat** [comment]
Hello,
We have done a fix for this issue. Please test it in latest version.
Thanks

**Comment by twillstudio** [comment]
Hi @sequoiaat , 

Im having the same issue than @willlaserlike .

"react-native": "^0.55.1",
"react-native-branch": "^2.3.3",

Could you please give me a solution ?
